api/odl.py

```diff
@@ -837,12 +860,19 @@ def _detail_for_elementtree_entry(cls, parser, entry_tag, feed_url=None, do_get=
                 if rel == "self":
                     odl_status_link = link_tag.attrib.get("href")
                     break
+            if not odl_status_link:
+                # TODO: This is a hack necessary because Feedbooks
+                # doesn't provide the status link yet. Once that's fixed,
+                # this link will be present with rel="self", just like
+                # in the unit tests.
+                odl_status_link = "https://license.feedbooks.net/copy/status/?uuid=" + identifier.replace("urn:uuid:", "")
```

Comment:
The need to do this `identifier.replace()` thing myself is the thing I dislike most about this hack, but hopefully we can get rid of it before release.